### ISSUE TITLE
Install R and the needed packages in docker

### ISSUE THREAD

**Issue by Thomas1e** [clemenshelm/chillbill-recognizer#244]
Installing R:
sudo apt-get install r-base r-recommended 

For now I need only two extra packages: 
run R (probably as root?) and type in the R-consol:
install.packages("e1071")
install.packages("jsonlite")

Alternative: I created an R script with these two lines. So you can run them via 

Rscript --vanilla installation.R

**Comment by tamacodechi** [comment]
Will we want this in our image of the recognizer or is this a separate one we will attach? 🐋 ✨

**Comment by clemenshelm** [comment]
The parent one (issue #254)

**Comment by Thomas1e** [comment]
The R version of the standard repository is 3.1.1 (2014-07-10) which is a little bit old. Maybe because of this some of packages can not be installed.

the dplyr package needs >= 3.1.2
jsonlite is very new (2017)

**Comment by Thomas1e** [comment]
With Pilars help I created a docker image and ran the container. It looks like the svm package "e1071" as well as the "jsonlite" packages work.

Unfortunately "lintr" as well as "dplyr" does not work yet. 
"lintr" is for linting the code. On my machine I had to install libcurl4-openssl-dev and  libssl-dev but I am using Ubuntu 17.04.

With "dplyr" we could make the code easier to read and faster because this package uses C++ for the data manipulation.
